src/test/kotlin/calculator/OperatorTest.kt

```diff
@@ -2,34 +2,42 @@ package calculator
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
 
 class OperatorTest {
     @Test()
-    fun `findBy 메서드는 전달받은 symbol과 일치하는 연산자 enum을 반환한다`() {
+    fun `전달받은 symbol과 일치하는 연산자 enum을 반환한다`() {
```

Comment:
```suggestion
    fun `symbol과 일치하는 연산자 enum을 반환한다`() {
```
"전달받은" 과 관련된 문구는 빼도 좋을 것 같아요 (이하 관련 내용 동일)